protos/feast/core/OnDemandFeatureView.proto

```diff
@@ -58,6 +58,9 @@ message OnDemandFeatureViewSpec {
 
     // Owner of the on demand feature view.
     string owner = 8;
+
+    // Backend used to handle on-demand transformations. One of python or pandas.
+    string mode = 9;
```

Comment:
probably should be enum right?

```
enum Mode {
  python = 0;
  pandas = 1;
}

message Msg {
  optional Mode mode = 1;
}
```

The annoying thing we'll have to do is map those from string to integers

Reply:
^ string -> enum